### PR TITLE
Add rpm packages

### PR DESCRIPTION
**Pull request by VJanKraemer**
Add RPM packages to the REPO
While doing this update the SIL Kit Version to 4.0.54-rc1 (NOT RELEASED YET)

### CODE REVIEW

**Comment by VJanKraemer** [comment]
To test the workflow, it is still named `package-debian.yml` but before merging I will rename it to `package-silkit.yml` . If we do it beforehand, we can unfortuanetly not test the dispatch workflow

**Comment by VJanKraemer** [comment]
Dispatch Test with artifacts (will vanish on 01.11 though) can be found here: https://github.com/vectorgrp/sil-kit-pkg/actions/runs/11607740388

File: scripts/silkit_deb.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+            # If we have '' empty strings in the command list, strange things happen in
+            # exec land
+            debuild_cmd = list(filter(lambda arg: arg != "", debuild_cmd))
+
+            logger.debug(f"CMD LIST: {debuild_cmd}")
+            logger.debug(f"Calling: {' '.join(debuild_cmd)}")
+            subprocess.run(debuild_cmd, check=True, cwd=self.build_info.work_dir / "sil-kit/")
+        except Exception as e:
+            raise RuntimeError(str(e))
```
Comment (VDanielEdwards) [inline scripts/silkit_deb.py:158]:
```suggestion
            raise RuntimeError("failed to build package") from e
```

Which adds the exception `e` to the exception context directly. See [the Python docs][1] under _Exception context_.

[1]: https://docs.python.org/3/library/exceptions.html#built-in-exceptions

Comment (VDanielEdwards) [reply scripts/silkit_deb.py:158]: Similarly for the other `raise` statements 😸

**Review by VDanielEdwards — APPROVED** [review]
The packages look good to me. I was able to install them on a throwaway container. Scripts look nice as well! 👍 Thank you!

File: tests/test_deb.py [hunk 2]
```diff
@@ -94,23 +111,33 @@ def run_test(build_dir):
...
-    test_dir = args.package_directory / 'tests'
+    test_dir = args.test_directory
     build_dir = test_dir / '_build'
-    build_test(test_dir)
-    run_test(build_dir)
+    build_test(test_dir, args.distro)
+    registry = run_registry()
+    try:
+        run_test(build_dir)
+    except:
```
Comment (VDanielEdwards) [inline tests/test_deb.py]: Is this `except` block deliberately empty?

**Comment by VJanKraemer** [comment]
New packages here (available for 06.11.24)
https://github.com/vectorgrp/sil-kit-pkg/actions/runs/11686289492

**Review by VDanielEdwards — APPROVED** [review]
Second test was successful as well! 👍 Thank you!

**Pull request merged**